configs/mattors/indexnet/README_zh-CN.md

```diff
@@ -17,15 +17,15 @@
 
 <br/>
 
-|                                    Method                                    |   SAD    |    MSE    |   GRAD   |   CONN   |                                                                                                                              Download                                                                                                                               |
+|                                    算法                                    |   SAD    |    MSE    |   GRAD   |   CONN   |                                                                                                                              下载                                                                                                                               |
 | :--------------------------------------------------------------------------: | :------: | :-------: | :------: | :------: | :-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------: |
-|                               M2O DINs (paper)                               |   45.8   |   0.013   |   25.9   | **43.7** |                                                                                                                                  -                                                                                                                                  |
-| [M2O DINs (our)](/configs/mattors/indexnet/indexnet_mobv2_1x16_78k_comp1k.py) | **45.6** | **0.012** | **25.5** |   44.8   | [model](https://download.openmmlab.com/mmediting/mattors/indexnet/indexnet_mobv2_1x16_78k_comp1k_SAD-45.6_20200618_173817-26dd258d.pth) \| [log](https://download.openmmlab.com/mmediting/mattors/indexnet/indexnet_mobv2_1x16_78k_comp1k_20200618_173817.log.json) |
+|                               M2O DINs (原文)                               |   45.8   |   0.013   |   25.9   | **43.7** |                                                                                                                                  -                                                                                                                                  |
+| [M2O DINs (复现)](/configs/mattors/indexnet/indexnet_mobv2_1x16_78k_comp1k.py) | **45.6** | **0.012** | **25.5** |   44.8   | [模型](https://download.openmmlab.com/mmediting/mattors/indexnet/indexnet_mobv2_1x16_78k_comp1k_SAD-45.6_20200618_173817-26dd258d.pth) \| [日志](https://download.openmmlab.com/mmediting/mattors/indexnet/indexnet_mobv2_1x16_78k_comp1k_20200618_173817.log.json) |
 
 > The performance of training (best performance) with different random seeds diverges in a large range. You may need to run several experiments for each setting to obtain the above performance.
```

Comment:
This one is a quote from the original paper or what? Should we translate this?

Reply:
Yes, original and reproduce.
I think we should translate this, and you?